Source/Eto.Mac/Eto.XamMac - net45.csproj

```diff
@@ -10,6 +10,7 @@
     <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
     <AssemblyName>Eto.XamMac</AssemblyName>
     <UseMSBuildEngine>True</UseMSBuildEngine>
+    <BaseIntermediateOutputPath>..\..\BuildOutput\obj\Eto.XamMac - net45</BaseIntermediateOutputPath>
```

Comment:
Not sure why these are being added to the csproj's, but they should be removed as they are set in Shared\Common.props